exonum-java-binding/tutorials/car-registry/car-registry-client/src/main/java/com/example/car/client/Config.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.car.client;
+
+/**
+ * The client application rudimentary configuration.
+ */
+final class Config {
+  static final String NODE_PUBLIC_API_HOST = "http://127.0.0.1:3000";
+  static final String NODE_JAVA_API_HOST = "http://127.0.0.1:7000";
```

Comment:
It's worth to add a few words in the tutorial why do we send tx to one endpoint and read the state from another

Reply:
But the present tutorial does not show requests to service at that level: both operations are hidden inside this CLI client that comes with the tutorial. That is, the user does not interact with these endpoints directly, but through a pre-configured client, therefore, we can't just add that there are two servers for different uses, we will have to explain a bit more.

Shall we add some explanation here for the curious?